### DISCUSSION TITLE
Plans for net7.0-android, net7.0-maccatalyst

### DISCUSSION THREAD

**Discussion by cosmin42** [microsoft/CsWinRT#1351] [Q&A]
Are there any plans for CSWinRT to support net7.0-android, net7.0-maccatalyst? Is there an issue that I can track?

**Comment by hez2010** [comment]
Why would you want the CsWinRT support for android and maccatalyst while those platforms don't have the support for WinRT at all?

**Comment by cosmin42** [comment]
I misunderstood that CSWinRT projects C++ in general, not just cppwinrt, I apologize about the question.
